packages/shared/src/components/sidebar/sections/ResourceSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement, useMemo } from 'react';
+import { ListIcon, SidebarMenuItem } from '../common';
+import { DocsIcon, FeedbackIcon, TerminalIcon } from '../../icons';
+import { docs, feedback, webappUrl } from '../../../lib/constants';
+import { Section } from '../Section';
+import { AlertColor, AlertDot } from '../../AlertDot';
+import { useChangelog } from '../../../hooks/useChangelog';
+import { SidebarSettingsFlags } from '../../../graphql/settings';
+import { SidebarSectionProps } from './common';
+
+export const ResourceSection = ({
+  isItemsButton,
+  ...defaultRenderSectionProps
+}: SidebarSectionProps): ReactElement => {
+  const changelog = useChangelog();
+  const menuItems: SidebarMenuItem[] = useMemo(() => {
+    return [
+      {
+        icon: () => <ListIcon Icon={() => <DocsIcon />} />,
+        title: 'Docs',
+        path: docs,
+        target: '_blank',
+      },
+      {
+        icon: () => <ListIcon Icon={() => <TerminalIcon />} />,
+        title: 'Changelog',
+        path: `${webappUrl}sources/daily_updates`,
+        rightIcon: () =>
+          changelog.isAvailable && (
+            <AlertDot className="-top-1 right-0" color={AlertColor.Cabbage} />
+          ),
```

Comment:
Do we still use this?

Reply:
Not 100% sure, just leaving it for now for a second

Reply:
👍